docs/content/annotations.md

```diff
@@ -238,6 +241,19 @@ allow {
 }
 ```
 
+#### Inlined Schema Format
+
+Schema definitions can be inlined by specifying the schema structure as a yaml map.
```

Comment:
Right! Will update the docs with the missing info 👍 .

Reply:
Regarding JSON, when the schema value is a string, the parser assumes it's a schema reference. The only way to inline a schema is by having the value be a map; which I think predicates it to be a YAML map. But I don't know the YAML spec like the back of my hand, so there might be some way to mix-and-match with JSON that I'm unaware of 😄 .

Reply:
YAML is supposedly a superset of JSON, so any JSON should be valid YAML too. 

(there are exceptions, but I don't think they'll be relevant in this case)

Reply:
So then perhaps it'll be confusing to say that an inlined schema can be both YAML or JSON?

Reply:
"JSON or YAML" would be good to me.

Reply:
The distinction would be 
```
# METADATA
# schemas:
# - input: {"type": "boolean"}
```
as compared to 
```
# METADATA
# schemas:
# - input: 
#      type: boolean
```
I presume? But since both are valid YAML, perhaps JSON doesn't need mentioning?

Reply:
I think it does, given how it's a _JSON schema_, whether expressed in YAML or JSON.